src/GenericAccessRequestForm/Constants/FormSpecificConstants.tsx

```diff
@@ -7,14 +8,24 @@ const StudyRegistrationAccessRequest = {
     'Please fill out this form to request and be approved for access to register your study with the HEAL Platform.',
   resultTitle: 'Your access request has been submitted!',
   resultSubtitle:
-    'Thank you for your submission. Requests take up to 1 business day to complete. You will be notified of the status.',
+  <React.Fragment> Thank you for your submission. Requests take up to 1 business day to complete.
+    {' '}You will be notified when approved.<br />
+    If you do not receive notification within 1 business day
+    {' '}of your request, please reach out to
+    {' '}<a href='mailto:heal-support@datacommons.io'>heal-support@datacommons.io</a>.
```

Comment:
This support email should not be hardcoded, it should comes from the `components.login.email` section from the portal config, see how the `Login` component is using it from portal config as a reference https://github.com/uc-cdis/data-portal/blob/master/src/Login/Login.jsx#L215-L220